mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -1522,7 +1522,7 @@ pub async fn enable_qrc20(mm: &MarketMakerIt, coin: &str, urls: &[&str], swap_co
 pub fn from_env_file(env: Vec<u8>) -> (Option<String>, Option<String>) {
     use regex::bytes::Regex;
     let (mut passphrase, mut userpass) = (None, None);
-    for cap in Regex::new(r"(?m)^(PASSPHRASE|USERPASS)=(\w[\w ]+)$")
+    for cap in Regex::new(r"\w+_(PASSPHRASE|USERPASS)=(\w[\w ]+)\n?")
```

Comment:
I guess this won't match the passphrase specified [here](https://github.com/KomodoPlatform/komodo-defi-framework/blob/main/docs/DEV_ENVIRONMENT.md#running-native-tests)
```env
PASSPHRASE=spice describe gravity federal blast come thank unfair canal monkey style afraid
```
We should probably update the documentation.
Also these will now need to be modified https://github.com/KomodoPlatform/komodo-defi-framework/blob/8fd29ecce9f32a9fb6b3bb3221a10d66dda05a67/mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs#L1198
https://github.com/KomodoPlatform/komodo-defi-framework/blob/8fd29ecce9f32a9fb6b3bb3221a10d66dda05a67/mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs#L3635

Reply:
I think we can fix the regex to match just 'PASSPHRASE' like that:
```^(\w+_|.{0})(PASSPHRASE|USERPASS)=(\w[\w ]+)\n?```

Reply:
Because of this `get_passphrase` statements used throughout tests
https://github.com/KomodoPlatform/komodo-defi-framework/blob/8fd29ecce9f32a9fb6b3bb3221a10d66dda05a67/mm2src/mm2_main/tests/mm2_tests/best_orders_tests.rs#L985-L986
I think using `BOB_PASSPHRASE`/`ALICE_PASSPHRASE` everywhere and updating documentation is the right approach.